devops/cloudformation.json

```diff
@@ -48,6 +48,16 @@
                 ]
             }
         },
+        {
+          "Type" : "AWS::EC2::InstanceConnectEndpoint",
```

Comment:
Does each service need one of these? I forget- there might be one already

Reply:
survey says you need one per subnet https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/connect-with-ec2-instance-connect-endpoint.html 

The deploy was (is) failing in master because we couldn't find a valid instance connect endpoint. That was the source of the mysterious SSH errors when running ansible.

Reply:
One per subnet though- so maybe we should create it manually in the console instead of in the CF for this project? i.e. it’s not really a project specific resource- see what I mean?

Reply:
I see what you're saying - maybe I could try to conditionally create the resource?, https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/conditions-section-structure.html 

Or alternatively just add a bash command with a better failure if it doesn't exist? That way the next person who sees that error wastes less time trying to make sense of it

Reply:
I think we should manually create this. I don't think we should link it to the project with cloudformation

Reply:
sounds good to me, I'll remove that part for now and keep the typo fix